.gitignore

```diff
@@ -6,9 +6,10 @@
 lucky
 .env
 /test-project/
-/node_modules
+node_modules
 /tmp/
 
 # Libraries don't need dependency lock
 # Dependencies will be locked in application that uses them
 /shard.lock
+yarn.lock
```

Comment:
Should this also be added to generated projects? https://github.com/luckyframework/lucky_cli/blob/master/src/generators/web.cr#L75 I've never really known what best practice for that sort of deal is.

Reply:
I've always seen yarn.lock committed to repos, so I think it's probably safer to leave it as is!

This was mostly to ensure that folks playing with these deps in the future don't accidentally push a yarn.lock, which could have some negative side effects like os-specific package sub-dependencies (if my limited knowledge serves me)